src/lib/shared/types/log.type.ts

```diff
@@ -0,0 +1,9 @@
+export interface ILog {
```

Comment:
Do not use `I` prefix in TS types. It's a C# convention that is rarely used in JS